### ISSUE TITLE
Tagalog translation work for Novavax being approved for ages 12 and over

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#46848]
We recently made updates to the [main COVID-19 page](https://www.va.gov/health-care/covid-19-vaccine/) that the Novavax vaccines is approved for children 12 and over. We need to get this new content translated in both Spanish and Tagalog. 

This ticket is to track the Tagalog pages. 

## Tasks
- [x] Send text to translators (Randi)
- [x] Update Tagalog page (Aliyah)
- [x] Copyedit Tagalog page (Randi)
- [x] Publish Tagalog page (Randi)

## Acceptance Criteria
- [x] New content is live on Tagalog page

**Comment by RLHecht** [comment]
@aliyahblackmore Here's the word doc back with the changes for this. Thanks![https://app.zenhub.com/files/133843125/b5f13e91-2fa2-4398-bd8b-9e29c6481673/download](https://app.zenhub.com/files/133843125/b5f13e91-2fa2-4398-bd8b-9e29c6481673/download)

**Comment by aliyahblackmore** [comment]
Thanks @RLHecht ! 

This is ready! I'll wait until we get back translation with the English content above to do another review. 
Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=16413

**Three notes:** 

- In what we received this time the numbers were spelled out, which is not consistent with what is on the page.

I pulled the number range (where necessary) from another part of the page and used that. Happy to change this back if there is a Tagalog style reason for this change, but it seems like using "12," or "17" as is, would be accurate. 
Example of this change: 12 hanggang 17 (12 to 17).

- In accordion **"Can eligible adolescents ages 12 to 17 get a COVID-19 vaccine at VA?"**  we don't have a translation for the remaining text after _**"Not all VA health facilities offer all 3 of these vaccines right now."**_  We got this back in the Spanish translation, though it wasn't highlighted/sent to them to translate. I took a look back at the past Tagalog translations that we've received and we've never had this translated.  I've left the words as is for the time being and will pass this to you via email, Randi, to send over to the translators! 

- Wondering in the accordion "Currently available COVID-19 vaccines" if we should remove the "(Johnson & Johnson)" that is in parenthesis, since it is written in the last bullet point? This would follow VA style guide rules for English and wondering if this would work in Tagalog and Spanish. I've left this as for the time being but flagging this as something that I noticed!
